<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Учет склада</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addPos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>650</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 42px;
border: none;
background: #3a7999;
color: #f2f2f2;
padding: 10px;
font-size: 18px;
border-radius: 5px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
    <property name="text">
     <string>Добавить позицию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delPos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 42px;
border: none;
background: #3a7999;
color: #f2f2f2;
padding: 10px;
font-size: 18px;
border-radius: 5px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
    <property name="text">
     <string>Удалить позицию</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>950</y>
      <width>1411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>960</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 42px;
border: none;
background: #3a7999;
color: #f2f2f2;
padding: 10px;
font-size: 18px;
border-radius: 5px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="swapPos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 42px;
border: none;
background: #3a7999;
color: #f2f2f2;
padding: 10px;
font-size: 18px;
border-radius: 5px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
    <property name="text">
     <string>Переложить позицию</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultTable">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>1411</width>
      <height>891</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
font-size: 24px;
/*height: 42px;
position: relative;
box-sizing: border-box;
ransition: all 500ms ease;*/</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="countOfPos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 42px;
border: none;
background: #3a7999;
color: #f2f2f2;
padding: 10px;
font-size: 18px;
border-radius: 5px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
height: 42px;
/*border: none;*/
padding: 10px;
font-size: 18px;
/*border-radius: 5px;*/
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddCont">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
height: 42px;
padding: 10px;
font-size: 18px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>600</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
height: 42px;
padding: 10px;
font-size: 18px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddProd">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>600</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
height: 42px;
padding: 10px;
font-size: 18px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 42px;
border: none;
color: black;
padding: 10px;
font-size: 18px;
border-radius: 5px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
    <property name="text">
     <string>Контейнер</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 42px;
border: none;
color: black;
padding: 10px;
font-size: 18px;
border-radius: 5px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
    <property name="text">
     <string>Наименование</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 42px;
border: none;
color: black;
padding: 10px;
font-size: 18px;
border-radius: 5px;
position: relative;
box-sizing: border-box;
transition: all 500ms ease; </string>
    </property>
    <property name="text">
     <string>Производитель</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
